README.md

```diff
@@ -1,70 +1,159 @@
+### Supported versions
 
-| Part | Repository |
-|------|------------|
-| Fusion | https://github.com/vaadin/fusion |
+- **21.0**
+- **14.7** LTS
+- **10.0** LTS
 
-## Vaadin components - building blocks for your web app
+For new users, the latest major version is recommended.  
+For production applications, the latest LTS is recommended.
 
-| Part | Repository |
-|------|------------|
-| Vaadin web components and themes | https://github.com/vaadin/web-components |
-| Java APIs for Vaadin components | https://github.com/vaadin/flow-components |
+<br>
 
-## Vaadin Collaboration Engine 
+## Web frameworks
+Free, open source frameworks for building modern web applications.
 
-The simplest way to build real-time collaboration into web apps, [check more details](https://vaadin.com/collaboration)
+<table width="100%">
+<tr>
+  <td width="50%">
 
-| Part | Repository |
-|------|------------|
-| Collaboration-engine | https://github.com/vaadin/collaboration-engine |
+  [**Vaadin Flow**](https://github.com/vaadin/flow)  
+  Build modern web apps 100% in Java
+  </td>
+  <td width="50%">
 
-## Vaadin Router
+  [**Vaadin Fusion**](https://github.com/vaadin/fusion)    
+  The modern front-end framework for Java
+  </td>
+</tr>
+</table>
 
-[Vaadin Router](https://github.com/vaadin/vaadin-router) is a client-side router for Web Components.
+<br>
+
+## UI components
+Free, open source UI components for creating high quality, accessible interfaces for business applications.
+
+<table width="100%">
+<tr>
+  <td width="50%">
+
+  [**Flow Components**](https://github.com/vaadin/flow-components)  
+  Java API for Vaadin components
+  </td>
+  <td width="50%">
+
+  [**Web Components**](https://github.com/vaadin/web-components)  
+  TypeScript/JavaScript/HTML API for Vaadin components
+  </td>
+</tr>
+</table>
+
+<br>
 
 ## Tools
+Commercial tools to speed up application development. [Pricing](https://vaadin.com/pricing)
+
+<table width="100%">
+<tr>
+  <td colspan="2">
+
+  [**Collaboration Engine**](https://github.com/vaadin/collaboration-engine)  
+  The simplest way to build real-time collaboration into web apps
+  </td>
+</tr>
+<tr>
+  <td width="50%">
+
+  [**Designer**](https://github.com/vaadin/designer)  
+  Design Vaadin UIs visually with drag'n'drop
+  </td>
+  <td width="50%">
+
+  [**TestBench**](https://github.com/vaadin/testbench)  
+  Visual regression tests for Flow-based UIs
+  </td>
+</tr>
+</table>
+
+<br>
 
-| Tool | Repository |
-|------|------------|
-| Designer | https://github.com/vaadin/designer |
-| TestBench | https://github.com/vaadin/testbench |
+## Integrations
+Plugins and add-ons for easy integration with 3rd party development tools.
 
-All tools are part of the [Vaadin Pro Subscription](https://vaadin.com/pricing).
+- [**Spring** add-on](https://github.com/vaadin/spring)
+- [**CDI** add-on](https://github.com/vaadin/cdi)
+- [**Maven** plugin](https://github.com/vaadin/maven-plugin)
```

Comment:
this link is for the maven-plugin for FW 7 and 8.

the code part of maven plugin has been moved into flow, which artifact name is `flow-maven-plugin`, when releasing with platform, we repackage everything with artifact name `vaadin-maven-plugin`

Reply:
Okay, thanks for the clarification!

I suppose there’s no place to link to then? Would be nice to show Maven support here somehow, but I’m not sure now how to best do that. But I suppose this is related to the Gradle support in a way, as they are both deps and build tools.

Reply:
Would this be good enough?

```suggestion
- [**Maven** plugin](https://github.com/vaadin/flow/tree/master/flow-plugins/flow-maven-plugin)
```

Reply:
I assume it’s at least better than linking to the old plugin 😄